lib/ansible/modules/cloud/google/gce.py

```diff
@@ -635,6 +643,7 @@ def main():
             ip_forward = dict(type='bool', default=False),
             external_ip=dict(default='ephemeral'),
             disk_auto_delete = dict(type='bool', default=True),
+            disk_size = dict(),
```

Comment:
Shouldn't this be typed? I'd rather raise a typerror during param validation if the user tries to give us `"foobar"` as a disk size than get an error from `create_volume`.